### ISSUE TITLE
Graph: JAWS is not announcing the text/data(g.V()) present under edit field control present under 'Graph' tab section of the page.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1057]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1247196

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 

Pre-Requisite:
Select "Gremlin(graph)" in API combo box while creating the Cosmos DB.

Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" from the left and select it.
Navigate  to "New Graph" menu item and select it.
Navigate through all the controls under Add Graph blade and enter details and selct ok button to create new 'Graph'.
Create 'Sample Graph under 'Gremlin  API' section and expand created graph control. 
Navigate through all the controls under "Gremlin API" Section and select "Graph" which appears after expending the items.
Navigate through all the controls under Graph.
Open JAWS and Navigate to  edit box control under 'Graph' tab section.
Observe the screen reader behavior.
Verify the issue.

Actual Result:
JAWS is not announcing the text/data(g.V()) present under edit field control present under 'Graph' tab section of the page. JAWS reads only 'input query edit contains text type in text' which is not correct.

Expected Result:
JAWS should announce the text/data(g.V()) present under edit field control present under 'Graph' tab section of the page. It should read the edit field data as 'input query edit contains text type in text g.V()'.


Note:
This issue does not exists with NVDA and NVDA.

**Comment by ato9000** [comment]
https://user-images.githubusercontent.com/8519407/132714837-e994df48-ccf3-46b4-ba14-b9bf62ac61b2.mp4
